src/bindings/python/src/pyopenvino/utils/utils.hpp

```diff
@@ -12,10 +12,15 @@
     // Generated by operator `==` inside pybind::handle.
     #pragma warning( push )
     #pragma warning( disable: 4996 )
+#elif defined(__GNUC__)
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
looks like deprecated API ov::Any is used, can we avoid it?

Reply:
No, there is another problem:
`/home/akupuros/openvino/openvino/src/core/include/openvino/core/any.hpp:449:20: warning: ‘bool pybind11::handle::operator==(const pybind11::handle&) const’ is deprecated: Use obj1.is(obj2) instead [-Wdeprecated-declarations]
  449 |         return rhs == lhs;`
  
Warning occurred at the junction of pybind11 and the templates inside ov::Any.